src/xbot/common/math/PIDManagerFactory.java

```diff
@@ -0,0 +1,45 @@
+package xbot.common.math;
+
+import com.google.inject.assistedinject.Assisted;
+
+public interface PIDManagerFactory {
```

Comment:
Agreed, and mostly just because PIDManager has a bajillion different constructors which we don't often do. That said for this kind of thing feels worth the initial investment of time on the CommonLib side to ease things at competition bot time for sure.